assets/prometheus-user-workload/federate-route.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Route
+metadata:
+  labels:
+    app.kubernetes.io/part-of: openshift-monitoring
+  name: prometheus-uwm-federate
```

Comment:
```suggestion
  name: prometheus-user-workload-federate
```

Reply:
@simonpasquier , Route with name `prometheus-user-workload-federate` causes the following error due to long name.

```
 Updating Prometheus-user-workload failed: reconciling UserWorkload federate Route failed: creating Route object failed: Route.route.openshift.io "prometheus-user-workload-federate" is invalid: spec.host: Invalid value: "prometheus-user-workload-federate-openshift-user-workload-monitoring.apps.ci-ln-53rm4lk-72292.origin-ci-int-gce.dev.rhcloud.com": must be no more than 63 characters
```

I guess we can stick to a simple name like `federare` as it is a sub domain of `openshift-user-workload-monitoring.apps.ci-ln-53rm4lk-72292.origin-ci-int-gce.dev.rhcloud.com`.